src/opera/parser/tosca/csar.py

```diff
@@ -97,109 +344,35 @@ def package_csar(self, output, service_template=None, csar_format="zip"):
                 meta_file.touch()
                 meta_file.write_text(content)
 
-                return shutil.make_archive(output, csar_format, tempdir)
-        except Exception as e:
-            raise ParseError("Error when creating CSAR: {}".format(e), self)
+                return shutil.make_archive(output_file, csar_format, tempdir)
+        except Exception:
+            import traceback
```

Comment:
Is this import here because it's only needed for errors? Wouldn't it be nicer to have all imports at the top anyway?

Reply:
Of course. I forgot to move it :)